common/djangoapps/student/models.py

```diff
@@ -292,7 +292,7 @@ def age(self):
         year_of_birth = self.year_of_birth
         year = datetime.now(UTC).year
         if year_of_birth is not None:
-            return year - year_of_birth
+            return year - year_of_birth - 1
```

Comment:
And if you can provide some comments on the code to explain why we like age before the birthday than the age after the birthday, that would be great